src/main/resources/config/application.yml

```diff
@@ -161,6 +161,8 @@ spring:
             client-secret: web_app
             scope: openid, profile, email, offline_access # last one for refresh tokens
 
+  jersey:
+    application-path: /scim
```

Comment:
This moves all of the JAX-RS SCIMple endpoints under /scim, and lets the _normal_ Spring Controllers work as expected:

Something like this is also needed when configuration Okta's SCIM simple with Basic Auth:

```yaml
spring:
  security:
    user:
      name: <username>
      password: <password>
```